src/Search.php

```diff
@@ -8163,7 +8163,21 @@ public static function showItem($type, $value, &$num, $row, $extraparam = '')
                 } else {
                     $value = preg_replace('/' . self::LBBR . '/', '<br>', $value);
                     $value = preg_replace('/' . self::LBHR . '/', '<hr>', $value);
-                    $out .= $value;
+                    // Print parent elements with less contrasted color
+                    if (preg_match('/ > /', $value)) {
```

Comment:
I guess it would be preferable here to validate that the displayed option is related to a `completename` field. Indeed, even if ` > ` is not something that people usually use in other fields, it may happen.

Reply:
Sure, feel free to modify the code to improve it. (I have no idea how to do what you suggest)